### ISSUE TITLE
Фильтр отображения заказов в панели управления

### ISSUE THREAD

**Issue by makovsv** [makovpv/Carzine#52]
В параметр отображения добавить опцию

Не показывать отмененные и выполненные заказы

![Image](https://github.com/makovpv/Carzine/assets/38849646/685d2a4e-a3ca-4e52-a1bd-19f984290192)

**Comment by makovpv** [comment]
да, только наверное лучше обратную опцию (галку) - "показать все"
а по-умолчанию галка выключена и отображаются заказы кроме отмененных и выполненных

**Comment by makovsv** [comment]
ок
